styles/styles.css

```diff
@@ -15,35 +15,653 @@ li {
   list-style-type: none;
 }
 
-/* base styles */
+/* ========================== Main styles ========================== */
 body {
   background: var(--primary);
   overflow-x: hidden;
   font-family: "Montserrat", sans-serif;
 }
 
-/* fonts */
-h1,
-h2,
-h3,
-h4 {
-  font-weight: normal;
-  line-height: 1.4em;
+.button {
+  display: inline-block;
+  background: #55b8f0;
+  color: var(--primary);
+  border-radius: 30px;
+  padding: 12px 0;
+  font-weight: 500;
+  font-size: 0.8em;
+  line-height: 12px;
+  text-align: center;
+  width: 216px;
 }
-p,
-a,
-li {
-  color: #9893d8;
-  line-height: 1.4em;
-  font-size: 1em;
-}
-h1,
-h3 {
-  font-size: 1.2em;
+
+.button:hover {
+  filter: brightness(95%);
 }
-h2 {
+
+.block-title {
   font-size: 1.6em;
+  font-weight: 600;
+  color: var(--primary-font-color);
 }
-h4 {
+
+.block-small-title {
   font-size: 1.1em;
-}
\ No newline at end of file
+  font-weight: 500;
+  color: var(--secondary-font-color);
+}
+
+.block-description {
+  font-size: 1em;
+  font-weight: 400;
+  color: var(--primary-font-color);
+}
+
+.container {
+  margin: 60px 15%;
```

Comment:
Recommend approach is to assign width of a container, not gutters.
https://getuikit.com/docs/container